_source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md

```diff
@@ -0,0 +1,50 @@
+
+[Cassidy Williams](https://cassidoo.co/): The director of Developer Experience at Netlify, Cassidy is an avid speaker and developer advocate. She’s been featured in Glamour Magazine's *35 Women Under 35 Changing the Tech Industry* and appears in *Big Dream*, a documentary about women who are breaking down barriers to follow their passion in science, math, computing, and engineering. 
+
+[Alyssa Miller](https://alyssasec.com/): As a hacker, researcher, and security advocate, Alyssa is committed to educating and inspiring others in the software development space. She’s spent her career speaking on technical and policy issues alike, advocating for privacy protection, and building trust with secure systems.
+
+[Milecia McGregor](https://www.mediavine.com/team/milecia-mcgregor/): Milecia is a senior software engineer, international tech speaker, and scientist who works with both hardware and software. She’s published her research on machine learning and robotics and started *Flipped Coding* in 2017 to help people learn web development with real-world projects.
+
+[Heather Downing](https://developer.okta.com/blog/authors/heather-downing/): Principal Developer Advocate at Okta, Heather is a passionate coder and international tech industry speaker from Kansas City, Missouri.  She’s been recognized as a Microsoft MVP and .NET Foundation member. Heather, a fierce advocate for women in tech, encourages those with non-traditional backgrounds to enter the world of software development and has built her reputation as an early adopter of new tech - without a degree.
+
+[Jessica Temporal](https://jtemporal.com/): Jessica is a senior developer advocate at Auth0 and is the co-founder and co-host of Pizza de Dados, the first (and most beloved) Brazilian podcast about data science. She’s an active participant in PyLadies, an international network for promoting and empowering women in technology focused on the Python open-source community.
+
+[ Kapehe](https://www.sanity.io/exchange/community/kapehe): Kapehe is a talented Sanity.io developer who finds joy in helping devs create, whether it’s a quick JAMstack project or a reliable content backend. She’s been working closely with the developer community and sharing her knowledge through her blog, YouTube channel, conference talks, and podcasts.
+
+## Register to Participate in Developer Day
+
+If you haven’t already registered for Developer Day, please head to our [Developer Day website](https://www.okta.com/developerday/) to sign up – it’s free and virtual, so you can tune in from wherever you are. And you can join our free, interactive learning sessions on August 25th by visiting the [Developer Day Labs livestream](https://developer-day.live/).
+
+I hope to see you there!
+
+
+Note: Recordings of the Developer Day sessions will be available on the event website for several weeks after the event, so don’t hesitate to register even if you can’t join us synchronously in real-time. 
```

Comment:
```suggestion
**Note:** Recordings of the Developer Day sessions will be available on the event website for several weeks after the event, so don’t hesitate to register even if you can’t join us synchronously in real-time. 
```

Add **bold** marker on "note".  if you want to add the blue background, you can stick a `>` in front to format that block as a quote